sdk/@launchdarkly/observability-node/src/api/Options.ts

```diff
@@ -20,7 +20,7 @@ export const ALL_CONSOLE_METHODS = [
 	'trace',
 	'warn',
 ] as const
-type ConsoleMethodsTuple = typeof ALL_CONSOLE_METHODS
+export type ConsoleMethodsTuple = typeof ALL_CONSOLE_METHODS
 export type ConsoleMethods = ConsoleMethodsTuple[number]
 
 /**
```

Comment:
Down on line 51, is the default otlp endpoint in the docs comment correct? It's listed as `https://otel.highlight.io:4318` but I think it might be `https://otel.observability.app.launchdarkly.com` or similar now?

Reply:
Yeah, I should double check the defaults that were part of the port. Ty for checking.